Initialtask/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.3"
+services:
+  mysql:
+    image: mysql:5.7
+    hostname: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: importTest
+      MYSQL_USER: dev
+      MYSQL_PASSWORD: dev
+      VIRTUAL_HOST: mysql.dev.loc
+    ports:
+      - "3306:3306"
```

Comment:
why do you make port visible to host?

Reply:
Mysql connection in bash-script don't working wihtout this option